pkg/e2e/operators/cloudingress/rhapi_lb.go

```diff
@@ -50,21 +74,28 @@ func getLBForService(h *helper.H, namespace string, service string) (string, err
 		// the LB wasn't created yet
 		return "", nil
 	}
+
+	if idtype == "ip" {
+		return ingressList[0].IP, nil
+	}
+
 	return ingressList[0].Hostname[0:32], nil
 }
 
-// testLBDeletion deletes the loadbalancer of rh-api service and ensures that cloud-ingress-operator recreates it
+// testLBDeletion deletes the load balancer of rh-api service and ensures that cloud-ingress-operator recreates it
 func testLBDeletion(h *helper.H) {
-	ginkgo.Context("rh-api-test", func() {
-		ginkgo.It("Manually deleted LB should be recreated", func() {
+	ginkgo.Context("rh-api-lb-test", func() {
+
 			if viper.GetString(config.CloudProvider.CloudProviderID) == "aws" {
```

Comment:
For this test to run, we should add a check for CCS, I have added a flag for the PR_Check:
OCM_CCS="true"

That will be the same flag that is used in our periodic testing. 
That can be grabbed by viper.GetBool("ocm.ccs") or viper.GetBool(CCS)